torch/_dynamo/config.py

```diff
@@ -267,6 +267,11 @@
 # [@compile_ignored: debug] Whether to report any guard failures (deprecated: does not do anything)
 report_guard_failures = True
 
+# For each guard failure, whether to print all failed checks, or only the first one
+# (note: dynamo at runtime does not run past the first failed check)
+# [@compile_ignored: debug]
+report_all_guard_failure_checks = False
```

Comment:
I think this should not be a config. Instead, it should be a logging artifact, so it can be operated remotely via TORCH_LOGS

Reply:
hmm, I guess instead of `report_all_guard_failure_checks` one could check `all_guard_failures_log.isEnabledFor(logging.DEBUG)`?